spec/models/project_spec.rb

```diff
@@ -248,4 +248,53 @@
       it_behaves_like 'respecting group assignment settings'
     end
   end
+
+  describe 'countable required_project_storage' do
+    describe '#required_storage' do
+      it 'counts collections' do
+        expect(project).to receive(:count_for).with(:required_project_storage).and_call_original
+        storage_hash = project.required_storage
+        expect(storage_hash).to eq({"attributes.attachments"=>0, :total=>0})
```

Comment:
Redundant curly braces around a hash parameter.<br>Surrounding space missing for operator '=>'.<br>Space inside { missing.<br>Space inside } missing.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.